contracts/interfaces/IAssessment.sol

```diff
@@ -0,0 +1,175 @@
+    uint8 MIN_VOTING_PERIOD_DAYS;
+    // The maximum number of days the users can vote on polls
+    uint8 MAX_VOTING_PERIOD_DAYS;
+    // Number of days the users must wait after a poll closes, to redeem payouts.
+    uint8 PAYOUT_COOLDOWN_DAYS;
+    // Percentage used to calculate assessment rewards (0-10000 i.e. double decimal precision)
+    uint16 REWARD_PERC;
+    // Percentage used to calculate potential impact of an incident
+    uint16 INCIDENT_IMPACT_ESTIMATE_PERC;
+    // Percentage out of 1 ETH, used to calculate a flat ETH deposit required for claim submission.
+    // If the claim is accepted, the user will receive the deposit back when the payout is redeemed.
+    uint16 CLAIM_ASSESSMENT_DEPOSIT_PERC;
+    // Percentage used to calculate the NXM deposit required for incident submission.
+    uint16 INCIDENT_ASSESSMENT_DEPOSIT_PERC;
+
+    uint168 _unused;
+  }
+
+  struct Stake {
+    uint96 amount;
+    uint104 rewardsWithdrawnUntilIndex;
+    uint16 fraudCount;
+    /*uint32 unused,*/
+  }
+
+  /*
+   *  Holds data for a vote belonging to an assessor.
+   *
+   *  The structure is used to keep track of user's votes. Each vote is used to determine
+   *  a user's share of rewards or to create a fraud resolution which excludes fraudulent votes
+   *  from the initial poll.
+   */
+  struct Vote {
+   // Identifier of the claim or incident
+    uint104 eventId;
+   // If the assessor votes to accept the event it's true otherwise it's false
+    bool accepted;
+   // Date and time when the vote was cast
+    uint32 timestamp;
+   // How many tokens were staked when the vote was cast
+    uint96 tokenWeight;
+   // Can be a claim or an incident (See EventType enum)
+    uint8 eventType;
+  }
+
+  struct Poll {
+    uint96 accepted;
+    uint96 denied;
+    uint32 start;
+    uint32 end;
+  }
+
+  /*
+   *  Holds the requested amount, NXM price, submission fee and other relevant details
+   *  such as parts of the corresponding cover details and the payout status.
+   *
+   *  This structure has snapshots of claim-time states that are considered moving targets
+   *  but also parts of cover details that reduce the need of external calls. Everything is fitted
+   *  in a single word that contains:
+   */
+  struct ClaimDetails {
+   // Amount requested as part of this claim up to the total cover amount
+    uint96 amount;
+   // The identifier of the cover on which this claim is submitted
+    uint32 coverId;
+   // Cover period represented as days, used to calculate rewards
+    uint16 coverPeriod;
+   // The asset which is expected at payout. E.g ETH, DAI (See Asset enum)
+    uint8 payoutAsset;
+   // The price (TWAP) of 1 NXM in the covered asset, at claim-time
+    uint80 nxmPriceSnapshot;
+   // A snapshot of CLAIM_ASSESSMENT_DEPOSIT_PERC if it is changed before the payout
+    uint16 assessmentDepositPerc;
+   // True when the payout is redeemed. Prevents further payouts on the claim.
```

Comment:
check if deposit is returned as well

Reply:
It happens at the same time with payout. If either of them fail, the whole tx fails.